Tests/ItemProviderTests.swift

```diff
@@ -281,7 +283,7 @@ class ItemProviderTests: XCTestCase {
             fixture(filePath: self.itemsPath, headers: nil)
         }
 
-        expiredProvider.provideItems(request: request) { (result: Result<[TestItem], ProviderError>) in
+        _ = expiredProvider.provideItems(request: request) { (result: Result<[TestItem], ProviderError>) in
```

Comment:
Changes to these are because of a warning due to the previous addition of the return of an `AnyCancellable` on the completion handler functions

Reply:
you could make it `@discardableResult` potentially, if the result isn't always used